tests/e2e/src/main-suite.e2e-spec.ts

```diff
@@ -129,6 +129,12 @@ describe('E2E Suite', function () {
     const validInputDescriptor = {
       constraints: {
         fields: [
+          {
+            path: ['$.id'],
+            filter: {
+              const: 'https://issuer.oidp.uscis.gov/credentials/83627465'
```

Comment:
Perhaps a minor point but this `.gov` URL as the id is a bit out of context here so better just to use a UUID, in my opinion
```suggestion
              const: 'urn:uuid:49f69fb8-f256-4b2e-b15d-c7ebec3a507e'
```